service/worker/deployment/fx.go

```diff
@@ -144,11 +149,11 @@ func (s *workerComponent) newDeploymentActivities(name namespace.Name, id namesp
 	}
 }
 
-// TODO Shivam - place holder for now but will initialize activity rate limits (if any) amongst other things
-func (s *workerComponent) newDeploymentSeriesActivities(name namespace.Name, id namespace.ID) *DeploymentSeriesActivities {
-	return &DeploymentSeriesActivities{
-		activityDeps: s.activityDeps,
-		namespace:    name,
-		namespaceID:  id,
-	}
-}
+// // TODO Shivam - place holder for now but will initialize activity rate limits (if any) amongst other things
+// func (s *workerComponent) newDeploymentSeriesActivities(name namespace.Name, id namespace.ID) *DeploymentSeriesActivities {
+// 	return &DeploymentSeriesActivities{
+// 		activityDeps: s.activityDeps,
+// 		namespace:    name,
+// 		namespaceID:  id,
+// 	}
+// }
```

Comment:
also cherry-picked from https://github.com/temporalio/temporal/pull/6864/commits/8c856b829c38f073f6056d24022cea8f74da9845 to get the server to build -- will remove